### ISSUE TITLE
RevEng: Unnecessary usings in entity class code

### ISSUE THREAD

**Issue by bricelam** [aspnet/EntityFramework#3229]
Entity classes include the following using directives.

```C#
using Microsoft.Data.Entity;
using Microsoft.Data.Entity.Metadata;
```

I can't think of any entity class code that would require them.

**Comment by lajones** [comment]
@rowanmiller Suggest pri0 is too high...

**Comment by divega** [comment]
@lajones agreed it is not critical, but we think it is small and we should just do it, so pri0 is to keep it in the radar for now.

**Comment by lajones** [comment]
Fixed with commit 1f3f1eb.
